spec/src/main/asciidoc/batch_runtime_spec.adoc

```diff
@@ -676,11 +676,11 @@ public interface Metric
 {
     public enum MetricType
     {
-        READ_COUNT_, _WRITE_COUNT_,
-        _COMMIT_COUNT_,
-        _ROLLBACK_COUNT_, _READ_SKIP_COUNT_, _PROCESS_SKIP_COUNT_,
-        _FILTER_COUNT_,
-        _WRITE_SKIPCOUNT
+        READ_COUNT, WRITE_COUNT,
+        COMMIT_COUNT,
+        ROLLBACK_COUNT, READ_SKIP_COUNT, PROCESS_SKIP_COUNT,
+        FILTER_COUNT,
+        WRITE_SKIPCOUNT
```

Comment:
```suggestion
        WRITE_SKIP_COUNT
```

This is what the released API has been using (see https://github.com/eclipse-ee4j/batch-api/blob/dc56a440c1442a8e71fd0fbf1b7298152e9e0393/api/src/main/java/jakarta/batch/runtime/Metric.java#L33) and  is also analogous to the names for the read/process skip counts.